### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by DikieDick**
# Add pull request template.

### Description
This PR adds a file to the GitHub repository that adds this template to future PR requests.

### Changes
Adds the `.github/pull_request_template.md` file to the repository. 

### Testing this PR
- Not relevant

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Title
```
Comment (pimotte) [inline .github/pull_request_template.md:1]:
```suggestion
```

I'd cut this out, because the title is already in the PR itself

**Review by pimotte — APPROVED** [review]
Good to merge after accepting or rejecting my suggestion:)

**Comment by DikieDick** [comment]
Ah, there is a newline now at the start and no newline at the end of the file. I will fix this.

**Comment by DikieDick** [comment]
Thanks for the review, I agree with your comment and have updated the file accordingly @pimotte.

**Pull request merged**